frontend/src/entry/overview/EntryRoleList.tsx

```diff
@@ -0,0 +1,23 @@
+import { Grid } from '@mui/material'
+import EntryCardEntityRoleRow from 'src/entry/overview/EntryCardEntityRoleRow'
+import { EntryInterface } from 'types/types'
+
+type EntryCardRoleDialogProps = {
+  entry: EntryInterface
+}
+
+export default function EntryRoleList({ entry }: EntryCardRoleDialogProps) {
```

Comment:
I don't see the value in splitting up into quite so many components, though it's definitely better to overuse components than underuse them! Consider combining this and `EntryCardRoleReview`:
```
export default function EntryRoleList({ entry }: EntryCardRoleDialogProps) {
  const rows = useMemo(
    () =>
      entry.collaborators.map((collaborator) => (
        <Fragment key={collaborator.entity}>
          <Grid item xs={6}>
            <Stack direction='row' alignItems='center' spacing={1}>
              <EntryCollaboratorIcon entryCollaborator={collaborator} />
              <EntryCollaboratorName entryCollaborator={collaborator} />
            </Stack>
          </Grid>
          <Grid item xs={6}>
            <EntryCardRolesChipSet entryCollaborator={collaborator} />
          </Grid>
        </Fragment>
      )),
    [entry.collaborators],
  )

  return (
    <Grid container spacing={2}>
      <Grid item xs={6}>
        Entity
      </Grid>
      <Grid item xs={6}>
        Roles
      </Grid>
      {rows}
    </Grid>
  )
}
```